### PR TITLE
[17.0][ADD] pos_default_shipping_date

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds default shipping date to today in the point of sale

image

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
@dalonsod i did it 😎

File: pos_default_shipping_date/static/description/index.html [hunk 1]
```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dalonsod) [inline pos_default_shipping_date/static/description/index.html]: Remove this as will fire a warning during Odoo installation (*)

(*) If you update OCA's maintainer tools it shouldn't be added anymore.

File: pos_default_shipping_date/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "summary": """
+        Adds default shipping date to today in the point of sale
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.1.0.0",
+    "category": "Extra Tools",
+    "website": "https://github.com/solvosci/slv-pos",
+    'depends': ['point_of_sale'],
+    'data': [] ,
```
Comment (dalonsod) [inline pos_default_shipping_date/__manifest__.py]: Remove empty section

File: pos_default_shipping_date/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+* Christian Santamaría <christian.santamaria@solvos.es>
```
Comment (dalonsod) [inline pos_default_shipping_date/readme/CONTRIBUTORS.rst]: Move this file and `DESCRIPTION.rst` to recommended `.md` extension, starting from v17. Once in `.md`, we need to escape these characters:
```suggestion
* Christian Santamaría \<christian.santamaria@solvos.es\>
```
Then, yo can re-create README files

**Review by dalonsod — APPROVED** [review]
:+1: please attend cosmetic changes and I'll merge it

**Comment by ChristianSantamaria** [comment]
@dalonsod Done :+1:

**Comment by dalonsod** [comment]
Could you install it at BasProd environment?

**Comment by ChristianSantamaria** [comment]
Done

**Pull request merged**